templates/pages/content.handlebars

```diff
@@ -42,7 +43,16 @@
 	  </div>
           <div class="flex-column">
             <div id="plugin-bug-report"></div>
-            <div id="plugin-rating"></div>
+            <bug-report
+              developer-institute="{{ collegeName }}"
+              context-info="Virtual Labs"
```

Comment:
context-info should have everything that is related to the context. Everything that is general should be independent attributes.
Think in terms of algodynamics, the attributes which would be applicable for both algodynamics and virtual labs should be attributes. The ones which would only appear in one of them, are part of context-info
Please discuss to understand what can be part of context and what can be generic attributes.